recipe/meta.yaml

```diff
@@ -39,19 +42,35 @@ source:
     # see https://bugreports.qt.io/browse/QTBUG-37209
     # fix phonon exports so they can be linked to
     - phonon_osx_exports.patch  # [osx]
+    # - moc_fix_includes.patch  # [win]
+    # pointer to avoid operations between pointers and numbers
+    - null_pointer_comparison.patch  # [osx]
+    # Add patch to use the option strip -x in osx 
+    - mac-strip.patch  # [osx]
 
 build:
-  number: 7  # [linux or win]
-  number: 8  # [osx]
-  skip: True  # [osx or (win and py>35)]
-  features:
-    - vc9  # [win and py27]
-    - vc10  # [win and py34]
-    - vc14  # [win and py>=35]
+  number: 1000
```

Comment:
This can be `9` instead of `1000` not that the migration is done the `1000` has no meaning.
(Also, if it was during the migration, it should've been `1009`.)